core/src/main/java/hudson/util/io/TarArchiver.java

```diff
@@ -59,7 +59,7 @@
     public void visitSymlink(File link, String target, String relativePath) throws IOException {
         TarArchiveEntry e = new TarArchiveEntry(relativePath, TarConstants.LF_SYMLINK);
         try {
-            int mode = IOUtils.mode(link);
```

Comment:
We generally try to not piss off thousands of users by changing the behavior of Jenkins for no real reason.